src/applications/find-forms/widgets/createInvalidPdfAlert.js

```diff
@@ -0,0 +1,74 @@
+// Dependencies.
+import React from 'react';
+import ReactDOM from 'react-dom';
+import * as Sentry from '@sentry/browser';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import { fetchFormsApi } from '../api';
+
+const InvalidFormDownload = () => (
+  <AlertBox
+    isVisible
+    status="error"
+    headline="This form link isn’t working"
+    content={
+      <>
+        We’re sorry, but the form you’re trying to download appears to have an
+        invalid link. Please{' '}
+        <a href="mailto:VaFormsManagers@va.gov">email the forms managers</a> for
+        help with this form.
+      </>
+    }
+  />
+);
+
+export async function onDownloadLinkClick(event) {
+  event.preventDefault();
+
+  const link = event.target;
+  const downloadUrl = link.href;
+  const formNumber = link.dataset.formNumber;
+
+  // Default to true in case we encounter an error
+  // determining validity through the API.
+  let formPdfIsValid = true;
+  let form = null;
+
+  try {
+    const forms = await fetchFormsApi(formNumber);
+    form = forms.find(f => f.id === formNumber);
+    formPdfIsValid = form?.attributes.validPdf;
+  } catch (err) {
+    // Todo
+  }
+
+  if (formPdfIsValid) {
+    link.removeEventListener('click', onDownloadLinkClick);
+    link.click();
+  } else {
+    Sentry.withScope(scope => {
+      scope.setExtra('form API response', form);
+      scope.setExtra('form number', formNumber);
+      scope.setExtra('download link (invalid)', downloadUrl);
+      Sentry.captureMessage('Find Forms - Form Detail - invalid PDF accessed');
+    });
+
+    const div = document.createElement('div');
+    const alertBox = <InvalidFormDownload />;
+
+    ReactDOM.render(alertBox, div);
+    link.parentNode.insertBefore(div, link);
```

Comment:
If the AlertBox changes its HTML at all, this code could fail I think 🤔 I wonder how we could make it more resilient, perhaps we should check to make sure `parentNode` exists before doing `insertBefore`? Or we could do some `?.` magic! 🧙

Reply:
> If the AlertBox changes its HTML at all

Just confirming - do you mean if the template changes its HTML at all? I think the AlertBox should be fine, right?

Reply:
Oh I don't think the AlertBox would target anything outside of it and change it, that's not what I meant. I just meant if the markup of the AlertBox were to ever change, it could affect this code unexpectedly and, while I don't foresee that happening anytime soon, it has happened in the past 😬